src/bin/release-please.ts

```diff
@@ -78,55 +67,30 @@ const argv = yargs
           describe: 'default branch to open release PR against',
           type: 'string',
         })
-        .option('path', {
-          describe: 'release from path other than root directory',
-          type: 'string',
-        })
         .option('monorepo-tags', {
           describe: 'include library name in tags and release branches',
           type: 'boolean',
           default: false,
         });
     },
     (argv: ReleasePROptions) => {
-      const rp = ReleasePRFactory.build(argv.releaseType, argv);
-      rp.run().catch(handleError);
+      factory.runCommand('release-pr', argv).catch(handleError);
     }
   )
   .command(
     'github-release',
     'create a GitHub release from a release PR',
     (yargs: YargsOptionsBuilder) => {
       yargs
-        .option('package-name', {
-          describe: 'name of package release is being minted for',
-        })
-        .option('repo-url', {
-          describe: 'GitHub URL to generate release for',
-          demand: true,
-        })
         .option('changelog-path', {
           default: 'CHANGELOG.md',
           describe: 'where can the CHANGELOG be found in the project?',
         })
-        .option('label', {
-          default: 'autorelease: pending',
-          describe: 'label to remove from release PR',
-        })
         .option('release-type', {
           describe: 'what type of repo is a release being created for?',
           choices: getReleaserNames(),
           default: 'node',
         })
```

Comment:
why did this `release-type` stay here? it's present below in the common options

Reply:
oversight, good catch.